clang/lib/CodeGen/BackendUtil.cpp

```diff
@@ -959,6 +960,10 @@ void EmitAssemblyHelper::EmitAssembly(BackendAction Action,
     PerModulePasses.add(createSPIRITTAnnotationsPass());
   }
 
+  // This pass should be always called for SYCL device code.
```

Comment:
nit: don't you think that the comment has a low value?

Reply:
Added another comment.